action.yml

```diff
@@ -15,9 +15,9 @@ inputs:
   coverageToken:
     description: token to send with coverage data
     required: false
-  ignoredUsers:
-    description: 'disable action to run on specified users PRs'
-    default: 'dependabot[bot]'
+  ignoreBots:
+    description: 'disable action to run on PRs created by bots'
```

Comment:
```suggestion
    description: 'disable action to run on pull requests created by bots'
```

Reply:
this is a public repository - try and keep slang or internally understood acronyms limited